physionet-django/events/templates/events/event_home.html

```diff
@@ -197,5 +197,28 @@ <h5 class="modal-title">Participants</h5>
        });
 
    });
+
+    $(document).ready(function() {
+        $('body').on('click', 'input[name="toggle-cohost-status"]', function(event) {
+            event.preventDefault(); // prevent the default action
+            console.log(event.target,'hello');
+            let checkbox = $(event.target);
+            let participant_id = checkbox.val();
+            let event_slug = checkbox.attr('event-slug');
+            let url = '/events/' + event_slug + '/participant/' + participant_id + '/toggle_cohost_status';
+
+            $.ajax({
+                url: url,
+                type: "GET",
+                success: function (data) {
+                  console.log(data);
+                  if(data[0].status===true){
+                    checkbox.prop('checked', !checkbox.prop('checked'));
+                  }
+                },
+            });
+        });
+    });
+
```

Comment:
Here i had to add the javascript code outside the template `event_entries.html` because if i add this code inside the `event_entries.html` the same block of javascript will get called multiple times(once for each event) because the `event_entries.html` is included multiple times inside the `event_home.html` template.

So adding the code directly in `event_home.html` template will work